.github/workflows/pull-request-ci-test.yml

```diff
@@ -20,6 +20,10 @@ jobs:
             sm_branch: "1.11-dev"
             spcomp_version: "1.11.x"
 
+          - sm_version: "1.12"
```

Comment:
I don't think `sm_version` is a key / value that's used anywhere in the workflow.  I think you wanted a corresponding `meta_branch`?